scss/components/_calendar.scss

```diff
@@ -0,0 +1,337 @@
+}
+
+// Body
+.fc-head {
+  border-top: 2px solid $neutral;
+}
+
+.fc-day-header {
+  color: $primary;
+  font-weight: normal;
+  padding-top: .5rem;
+}
+
+.fc-content-skeleton {
+  height: 100%;
+
+  table {
+    height: 100%;
+  }
+}
+
+.fc-week {
+  td:not(.fc-day-number) {
+    border-width: 0 2px;
+    border-style: solid;
+    border-color: $neutral;
+    border-top: none;
+  }
+
+  &:last-child {
+    border-bottom: 2px solid $neutral;
+  }
+}
+
+.fc-day-number {
+  border: 2px solid $neutral;
+  border-bottom: none;
+  color: $primary;
+  font-weight: bold;
+  padding: .5rem 1rem;
+}
+
+.fc-today {
+  background-color: $neutral;
+}
+
+.fc-event-container {
+  padding: 2px;
+  vertical-align: top;
+}
+
+.fc-content {
+  @include u-truncate();
+  color: $primary;
+  font-size: 1.4rem;
+  line-height: 1.2;
+  padding: 4px;
+}
+
+.fc-limited {
+  display: none;
+}
+
+// Events
+.fc--allday {
+  .fc-content {
+    background: $beige;
+    border-radius: 4px;
+    color: $cream;
+  }
+}
+
+.fc--election {
+  color: $cal-election;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-election;
+    }
+  }
+}
+
+.fc--deadline {
+  color: $cal-deadline;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-deadline;
+    }
+  }
+}
+
+.fc--outreach {
+ color: $cal-outreach;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-outreach;
+    }
+  }
+}
+
+.fc--rules {
+ color: $cal-rules;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-rules;
+    }
+  }
+}
+
+.fc--meeting {
+  color: $cal-meeting;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-meeting;
+    }
+  }
+}
+
+.fc--other {
+  color: $cal-other;
+
+  &.fc--allday {
+    .fc-content {
+      background: $cal-other;
+    }
+  }
+}
+
+// Week view
+.fc-slats {
+  tr:not(.fc-minor) {
+    border: 2px solid $neutral;
+    border-bottom: none;
+  }
+}
+
+// Event details
+.calendar__details {
+  font-size: 1.4rem;
+  margin-top: 1rem;
+  padding: 2rem;
+  text-align: left;
+
+  .button--close--primary {
+    position: absolute;
+    top: 0;
+    right: 0;
+  }
+}
+
+.fc-popup {
+  background: $cream;
+  border: 2px solid $neutral;
+  position: absolute;
+  max-width: 20rem;
+
+  .button {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 3rem;
+    height: 3rem;
+  }
+}
+
+.fc-popup__header {
+  background: $neutral;
+  color: $primary;
+  font-weight: bold;
+  padding: 4px;
+  position: relative;
+}
+
+// Week view
+.fc-time-grid {
+  .fc-content-skeleton {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+}
+
+// List view
+.ev-list {
+  margin-bottom: 1rem;
+  h5 {
+    margin: 0;
+  }
+}
+
+// Copied from fullcalendar.io
+.fc-unthemed .fc-row,
+.fc-unthemed .fc-popover {
+  border-color: #ddd;
+}
+
+.fc-unthemed .fc-popover {
+  background-color: #fff;
+}
+
+.fc-unthemed .fc-divider,
+.fc-unthemed .fc-popover .fc-header {
+  background: #eee;
+}
+
+.fc-unthemed .fc-popover .fc-header .fc-close {
```

Comment:
Merge rule `.fc-unthemed .fc-popover .fc-header .fc-close` with rule on line 262<br>Selector should have depth of applicability no greater than 3, but was 4